conf/simulator/jsbsim/aircraft/rotwing3.xml

```diff
@@ -827,6 +827,7 @@
           <value>47.9</value> <!-- Conversion to pascals -->
           <value>0.0151</value> <!-- CD x Area (m^2) -->
           <value>0.224808943</value> <!-- N to LBS -->
+          <value>3</value>
```

Comment:
It is a bit strange to add an extra parameter rather than multiplying the CD coef by 3. In addition, without comments it might be a problem in the future to understand where this value is coming from. I recommend to change the CDxArea by the proper value, unless there is a good reason to do otherwise.

Reply:
Changed!